files/zh-cn/web/javascript/reference/statements/import/index.md

```diff
@@ -32,7 +32,7 @@ var promise = import("module-name");//这是一个处于第三阶段的提案。
 - `defaultExport`
   - : 导入模块的默认导出接口的引用名。
 - `module-name`
-  - : 要导入的模块。通常是包含目标模块的 `.js` 文件的相对或绝对路径名，可以不包括 `.js` 扩展名。某些特定的打包工具可能允许或需要使用扩展或依赖文件，它会检查比对你的运行环境。只允许单引号和双引号的字符串。
+  - : 要导入的模块。通常是包含目标模块的 `.js` 文件的相对或绝对路径名，可以不包括 `.js` 扩展名。某些特定的打包工具可能允许不添加扩展名的导入模块，它会检查比对你的运行环境。只允许单引号和双引号的字符串。
```

Comment:
Hi @smallnine9, could you retranslate this paragraph (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).

![image](https://user-images.githubusercontent.com/15844309/202850316-811887ec-bd85-4f91-919b-ccd8d11c61a5.png)

Reply:
```suggestion
  - : 要导入的模块。访问修饰符与主机相关。其通常是包含目标模块的 `.js` 文件的相对或绝对路径名。在 Node 中，通常在引用 `node_modules` 中的包时使用不添加扩展名的导入。某些特定的打包工具可能允许不添加扩展名的导入模块方式，请检查你的环境。模块名只允许单引号和双引号的字符串。
```

Reply:
sorry miss the comment, and I think your translations are good